AzureTable/Microsoft.DataTransfer.AzureTable/RemoteLogging/RemoteLogging.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.CosmosDB;
+using Microsoft.Azure.CosmosDB.Table;
+using Microsoft.Azure.Storage;
+
+namespace Microsoft.DataTransfer.AzureTable.RemoteLogging
+{
+    /// <summary>
+    /// Class to support remote logging in CosmosDB Tables
+    /// </summary>
+    public sealed class RemoteLogging : IRemoteLogging
+    {
+        private static CloudTable migrationLogger;
+        private readonly int tableThroughput = 20000;
+        private readonly string tableName = "migrationLogs";
+
+        /// <summary>
+        /// Remote logger constructor
+        /// </summary>
+        /// <param name="storageAccount"></param>
+        /// <param name="connectionPolicy"></param>
+        public RemoteLogging(CloudStorageAccount storageAccount, TableConnectionPolicy connectionPolicy)
+        {
+            CloudTableClient tcMigrationLogger = storageAccount.CreateCloudTableClient(connectionPolicy: connectionPolicy);
+            migrationLogger = tcMigrationLogger.GetTableReference(tableName);
+        }
+
+        /// <summary>
+        /// Create a Table if it does not exists already
```

Comment:
"does not exist"

Reply:
N/A here .. applied feedback to the interface in new location Core project.